src/com/facebook/buck/jvm/kotlin/KotlinTestDescription.java

```diff
@@ -168,27 +164,5 @@ public void findDepsForTargetFromConstructorArgs(
   @BuckStyleImmutable
   @Value.Immutable
   interface AbstractKotlinTestDescriptionArg
-      extends HasContacts, HasTestTimeout, KotlinLibraryDescription.CoreArg {
-    ImmutableList<String> getVmArgs();
-
-    Optional<TestType> getTestType();
-
-    Optional<Level> getStdErrLogLevel();
-
-    Optional<Level> getStdOutLogLevel();
-
-    Optional<Long> getTestCaseTimeoutMs();
-
-    ImmutableMap<String, String> getEnv();
-
-    @Value.Default
-    default boolean getRunTestSeparately() {
-      return false;
-    }
-
-    @Value.Default
-    default ForkMode getForkMode() {
-      return ForkMode.NONE;
-    }
-  }
+      extends KotlinLibraryDescription.CoreArg, JavaTestDescription.CoreArg {}
```

Comment:
by the way, you are bringing more by using JavaTestDescription.CoreArg, it inherits `JavaLibraryDescription.CoreArg`. The proper way is to create an abstract JvmBasedTestArg<T> where T is the type of library. You would be able to use it like this: 

    extends KotlinLibraryDescription.CoreArg, JvmBasedTestArg<KotlinLibraryDescription.CoreArg> {}